modules/oe_content_call_proposals/config/install/field.storage.node.oe_call_proposals_contact.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - entity_reference_revisions
+    - node
+    - oe_content_entity_contact
+id: node.oe_call_proposals_contact
+field_name: oe_call_proposals_contact
+entity_type: node
+type: entity_reference_revisions
+settings:
+  target_type: oe_contact
+module: entity_reference_revisions
+locked: false
+cardinality: -1
```

Comment:
There is a comment in https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-436: "Contacts / oe_call_proposals_contact: The cardinality and bundle reference are not defined. We have set them as "cardinality: 1" and "reference: Contact entity General bundle"."

Reply:
I included this in my review already above.